content/articles/notifier.md

```diff
@@ -170,13 +170,7 @@ func (l *Notifier) runOnce(ctx context.Context) error {
     l.mu.RLock()
     defer l.mu.RUnlock()
 
-    subs := l.subscriptions[notification.Channel]
-
-    if len(subs) < 1 {
-        return nil
-    }
-
-    for _, sub := range subs {
+    for _, sub := range l.subscriptions[notification.Channel] {
```

Comment:
```suggestion
    // Notify subscribers (this is a no-op if no subs/empty slice).
    for _, sub := range l.subscriptions[notification.Channel] {
```
Opted for this since it's more accurate (thought range does gracefully handle both nil and empty slices and maps).